llvm/lib/Target/EVM/EVMFinalizeStackFrames.cpp

```diff
@@ -171,10 +171,13 @@ bool EVMFinalizeStackFrames::runOnModule(Module &M) {
 
   // Check if it is valid to replace frame indices.
   if (TotalStackSize > 0) {
-    if (TotalStackSize > StackRegionSize)
-      report_fatal_error("Total stack size: " + Twine(TotalStackSize) +
-                         " is larger than the allocated stack region size: " +
-                         Twine(StackRegionSize));
+    if (TotalStackSize > StackRegionSize) {
```

Comment:
Could we unify the terminology? We previously introduced term StackRegion and now we start using StackArea?